12-spatial-cv.Rmd

```diff
@@ -6,21 +6,25 @@ This chapter assumes proficiency with geographic data analysis\index{geographic
 A familiarity with generalized linear models (GLM)\index{GLM} and machine learning\index{machine learning} is highly recommended [for example from @zuur_mixed_2009;@james_introduction_2013].
 
 The chapter uses the following packages:^[
-Package **kernlab**, **pROC**, **RSAGA**\index{RSAGA (package)} and **spDataLarge** must also be installed although these do not need to be attached.
+Packages **GGally**, **lgr**, **kernlab**, **ml3measures**, **paradox**, **pROC**, **progressr** and **spDataLarge** must also be installed although these do not need to be attached.
 ]
 
-```{r 11-spatial-cv-1, message=FALSE }
+```{r 12-spatial-cv-1, message=FALSE }
 library(sf)
-library(raster)
+library(terra)
 library(dplyr)
 ```
 
-```{r 11-spatial-cv-1-2, eval=FALSE}
-library(mlr)
-library(parallelMap)
+```{r 12-spatial-cv-1-2, eval=FALSE}
+library(future)
+library(mlr3)
+library(mlr3learners)
+library(mlr3extralearners)
```

Comment:
I don't think we need to load this and potentially others. Worth adding a footnote, like in the transport chapter?

Reply:
Like this:

https://github.com/Robinlovelace/geocompr/blob/9e59777d098fd1b3cb06f78d84fa9a8409b30da5/13-transport.Rmd#L7-L9

Reply:
Pls see my comment below. The idea is good but probably won't work here.